data/vibe/data/json.d

```diff
@@ -2010,7 +2016,6 @@ void writeJsonString(R, bool pretty = false)(ref R dst, in Json json, size_t lev
 			dst.put('{');
 			bool first = true;
 			foreach (string k, ref const Json e; json.byKeyValue) {
-				if( e.type == Json.Type.undefined ) continue;
```

Comment:
I've removed this because it will not serialize the attribute at all and I think, that this should be handled by `@optional` attribute.